### ISSUE TITLE
Jupyterlab message upon startup

### ISSUE THREAD

**Issue by martindurant** [dask/dask-tutorial-infrastructure#4]
```
@jupyter-widgets/jupyterlab-manager needs to be included in build
jupyterlab_bokeh needs to be included in build
@jupyterlab/hub-extension changed from file:../extensions/jupyterlab-hub-extension-0.8.2.tgz to file:../extensions/jupyterlab-hub-extension-0.9.0.tgz
```
(presumably some versions were updated in the image?)

**Comment by TomAugspurger** [comment]
I've occasionally noticed an error in the `jupyter labextension` step, but IIRC the build doesn't actually fail. May be misremembering / unrelated though.

**Comment by martindurant** [comment]
Things worked fine, but the message popping up in the interface was odd - I just ignored it.

**Comment by TomAugspurger** [comment]
I *think* anything actually using the extension won't work correctly
though. If it was during jupyterwidgets, e.g., the `KubeCluster` repr would
be broken.

**Comment by martindurant** [comment]
@TomAugspurger , I think you have the appropriate credentials to push new images, it would be worthwhile rebuilding - and may as well update versions while at it.

**Comment by TomAugspurger** [comment]
See https://github.com/dask/dask-tutorial-infrastructure/pull/6 updating the image builders.

Once that's merged, you should be able to try it out.

**Comment by martindurant** [comment]
Seems to have been fixed by #7 (or perhaps any image rebuild would have done it)
